whitelist.yml

```diff
@@ -2,18 +2,31 @@
 ap-southeast:
   - 52.62.30.119
   - 52.62.3.168
+  - 54.66.51.34
+  - 54.79.207.8
+  - 52.65.195.15
+  - 52.62.240.64
+  - 54.252.43.78
+  - 54.79.124.27
 eu-central:
-  - 52.29.64.156
-  - 18.196.195.93
+  - 185.17.255.77
 eu-west:
   - 18.134.20.192
-frankfurt:
-  - 185.17.255.77
+  - 18.168.127.171
+  - 35.178.164.196
+  - 18.169.73.62
 us-east:
   - 52.23.148.198
   - 52.4.18.170
-nz:
-  - 103.16.181.211
+  - 107.21.125.147
+  - 54.85.120.50
+  - 52.45.11.42
+  - 54.84.197.145
+  - 52.204.99.5
+  - 18.213.233.108
```

Comment:
Where does this IP come from?
18.213.233.108

Reply:
that is one of the new Staging USA IP addresses